rest/route/user.go

```diff
@@ -86,6 +86,61 @@ func (h *userSettingsGetHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(apiSettings)
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// GET /rest/v2/users/{user_id}
+
+type getUserHandler struct {
+	userId string
```

Comment:
Can we call this `userDisplayName`? This isn't using the user ID.

Reply:
Didn't change since I switched to ID.